README.md

```diff
@@ -56,7 +70,8 @@ Convert the following create method in Mongoose to ActiveRecord.
 ```
 
 ```rb
-
+author = Author.create(:name => body.name)
```

Comment:
In this case we would need to use `params` instead of `body`. So something like `@author = Author.create!(name: params[:name])`.